docs/build/install.md

```diff
@@ -62,33 +63,33 @@ subql-node <command>
 ```
 
 ::: info Note
-If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure. 
+If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure.
 :::
 
 ## Install @subql/query
 
-The SubQuery query library provides a service that allows you to query your project in a "playground" environment via your browser. 
+The SubQuery query library provides a service that allows you to query your project in a "playground" environment via your browser.
 
 Install SubQuery query globally on your terminal by using Yarn or NPM:
 
 <CodeGroup>
   <CodeGroupItem title="NPM">
 
-  ```bash
-  npm install -g @subql/query
-  ```
+```bash
+npm install -g @subql/query
+```
 
   </CodeGroupItem>
 
   <CodeGroupItem title="YARN" active>
 
-  ```shell
-  yarn global add @subql/query
-  ```
+```shell
+yarn global add @subql/query
+```
 
   </CodeGroupItem>
 </CodeGroup>
 
 ::: info Note
-If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step also. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure. 
-:::
\ No newline at end of file
+If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step also. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure.
```

Comment:
```suggestion
If you are using Docker or hosting your project in SubQuery Projects, you can skip this step also. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure.
```